src/ddmd/cond.d

```diff
@@ -64,6 +67,227 @@ extern (C++) abstract class Condition : RootObject
 }
 
 /***********************************************************
+ * This class implements common functionality for StaticForeachDeclaration and StaticForeachStatement
+ */
+
+immutable StaticForeach_tupleFieldName = "tuple"; // used in lowering
```

Comment:
1. The Ddoc comment applies to the next declaration, which would be the above rather than the intended `StaticForeach`.

2. It looks like it belongs as a static field of `StaticForeach` anyway, which would resolve (1).